plugins/source/aws/resources/services/ram/resource_share_invitations.go

```diff
@@ -25,6 +25,14 @@ func ResourceShareInvitations() *schema.Table {
 					PrimaryKey: true,
 				},
 			},
+			{
+				Name:     "receiver_combined",
```

Comment:
Is this controversial? The reasoning is in the PR desc.

Reply:
It's OK, we did something similar in #6772